docs/pages/base-theme.css

```
@@ -0,0 +1,981 @@
+:root {
+  --cyan-50: #e9f9fc;
+  --cyan-100: #c1ecf4;
+  --cyan-200: #99dae5;
+  --cyan-300: #66bbca;
+  --cyan-400: #1f91a5;
+  --cyan-500: #0d535f;
+  --cyan-600: #094651;
+  --cyan-700: #063b44;
+  --cyan-800: #042f37;
+  --cyan-900: #022126;
+
+  --grey-50: #f3f6f9;
+  --grey-100: #e5eaf2;
+  --grey-200: #dae2ed;
+  --grey-300: #c7d0dd;
+  --grey-400: #b0b8c4;
+  --grey-500: #9da8b7;
+  --grey-600: #6b7a90;
+  --grey-700: #434d5b;
+  --grey-800: #303740;
+  --grey-900: #1c2025;
+}
+
+.GalleryBadge {
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+  font-variant: tabular-nums;
+  list-style: none;
+  font-family: IBM Plex Sans, sans-serif;
+  position: relative;
+  display: inline-block;
+  line-height: 1;
+}
+
+.GalleryBadge-badge {
+  box-sizing: border-box;
+  z-index: auto;
+  position: absolute;
+  top: 0;
+  right: 0;
+  min-width: 22px;
+  height: 22px;
+  padding: 0 6px;
+  color: #fff;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 22px;
+  white-space: nowrap;
+  text-align: center;
+  border-radius: 12px;
+  background: var(--cyan-500);
+  box-shadow: 0px 4px 8px var(--GalleryBadge-badge-boxShadow-color, var(--grey-300));
+  transform: translate(50%, -50%);
+  transform-origin: 100% 0;
+}
+
+.GalleryBadge-content {
+  width: 40px;
+  height: 40px;
+  border-radius: 12px;
+  background: var(--GalleryBadge-content-background-color, var(--grey-300));
+  display: inline-block;
+  vertical-align: middle;
+}
+
+@media (prefers-color-scheme: dark) {
+  .GalleryBadge {
+    --GalleryBadge-badge-boxShadow-color: var(--grey-900);
+    --GalleryBadge-content-background-color: var(--grey-400);
+  }
+}
+
+.GalleryButton {
+  font-family: IBM Plex Sans, sans-serif;
+  font-weight: 600;
+  font-size: 0.875rem;
+  line-height: 1.5;
+  background-color: var(--cyan-500);
+  padding: 8px 16px;
+  border-radius: 8px;
+  color: white;
+  transition: all 150ms ease;
+  cursor: pointer;
+  border: 1px solid var(--cyan-500);
+  box-shadow: 0 2px 4px var(--GalleryButton-boxShadow-color, rgba(13, 84, 99, 0.5)),
+    inset 0 1.5px 1px var(--cyan-400), inset 0 -2px 1px var(--cyan-600);
+}
+.GalleryButton:hover {
+  background-color: var(--cyan-600);
+}
+.GalleryButton:active {
+  background-color: var(--cyan-700);
+  box-shadow: none;
+}
+.GalleryButton:focus-visible {
+  box-shadow: 0 0 0 4px var(--GalleryButton-focusVisible-boxShadow-color, var(--cyan-200));
+  outline: none;
+}
+.GalleryButton:disabled {
```

Doesn't work with custom hosts

```suggestion
.GalleryButton.Mui-disabled {
```